packages/api/tests/endpoints/test-executions.js

```diff
@@ -152,6 +167,21 @@ test.before(async (t) => {
 
   jwtAuthToken = await createFakeJwtAuthToken({ accessTokenModel, username });
 
+  // Create AsyncOperation in Dynamo and Postgres
+  const testAsyncOperation = fakeAsyncOperationFactory({ id: uuidv4() });
+  t.context.testAsyncOperation = await asyncOperationModel.create(
+    testAsyncOperation
+  );
+
+  const testPgAsyncOperation = fakeAsyncOperationRecordFactory({
```

Comment:
Instead of creating a new, separate PG fixture, I would transform `testAsyncOperation` into an async operation for PG using `translateApiAsyncOperationToPostgresAsyncOperation`